src/pages/public_registration/public_files/FilesLandingPageAccordion.tsx

```diff
@@ -33,7 +33,7 @@ export const FilesLandingPageAccordion = ({ registration }: PublicRegistrationCo
       defaultExpanded
       heading={
         showRegistrationHasFilesAwaitingApproval ? (
-          <Box component="span" sx={{ bgcolor: 'secondary.light', p: '0.25rem' }}>
+          <Box component="span" sx={{ background: '#FEFBF3', p: '0.25rem' }}>
```

Comment:
Litt fyfy med inline farge her, men detter er også noe vi bare har satt for at det skal se ok ut, så tillater meg selv å se mellom fingrene her. Samme for alternatingColors for tabell. Tipper det kommer en dag ikke veldig langt frem i tid begge disse er borte uanZ